source/Body.h

```diff
@@ -106,6 +110,9 @@ class Body {
 	float scale = 1.f;
 
 	double alpha = 1.;
+	// The maximum distance at which the body is visible, and at which it becomes invisible again.
```

Comment:
I guess this is for the second variable? If yes, the first one could also use a comment.

Reply:
I think this covers both fields?

Reply:
It does. Two separate comments might be nice, but I don't think it's a big blocker.